src/nats.h

```diff
@@ -7813,6 +7833,11 @@ struct micro_endpoint_info_s
      */
     const char *Subject;
 
+    /**
+     * @brief Endpoint's queue group, or omitted if no queue group is used.
```

Comment:
Is it "omitted if no queue group is used" or if no "override"? Meaning, if I don't set something here, are we going to use the default, unless the NoQueueGroup is set?

Reply:
Clarified in the comment (the actual value, or "nil")